### PR TITLE
Add cookie-cutter-jaeger

### PR DESCRIPTION
**Pull request by mag415**

### CODE REVIEW

File: packages/jaeger/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@walmartlabs/cookie-cutter-jaeger",
+    "version": "1.0.0-beta.1",
```
Comment (sklose) [inline packages/jaeger/package.json:3]: can you make this alpha?

File: yarn.lock [hunk 2]
```diff
@@ -11044,7 +11134,12 @@ xmlcreate@^1.0.1:
   resolved "https://registry.yarnpkg.com/xmlcreate/-/xmlcreate-1.0.2.tgz#fa6bf762a60a413fb3dd8f4b03c5b269238d308f"
   integrity sha1-+mv3YqYKQT+z3Y9LA8WyaSONMI8=
 
-xtend@^4.0.0, xtend@~4.0.1:
+xorshift@^0.2.0:
+  version "0.2.1"
+  resolved "https://pkgs.dev.azure.com/jet-tfs/_packaging/KeplerTool/npm/registry/xorshift/-/xorshift-0.2.1.tgz#fcd82267e9351c13f0fb9c73307f25331d29c63a"
```
Comment (sklose) [inline yarn.lock:11139]: this got to be the public registry url

File: packages/jaeger/package.json (hunk 1 above)
Comment (mag415) [reply packages/jaeger/package.json]: updated

File: yarn.lock (hunk 2 above)
Comment (mag415) [reply yarn.lock]: updated

**Pull request merged**